core/block/import/markdown/blockconverter.go

```diff
@@ -163,12 +163,16 @@ func (m *mdConverter) processCSVFileLink(block *model.Block, files map[string]*F
 	files[link].HasInboundLinks = true
 }
 
-func (m *mdConverter) processFileBlock(block *model.Block, files map[string]*FileInfo) {
+func (m *mdConverter) processFileBlock(block *model.Block, files map[string]io.ReadCloser, importPath string) {
 	if f := block.GetFile(); f != nil {
 		if block.Id == "" {
 			block.Id = bson.NewObjectId().Hex()
 		}
-		m.createFile(f, block.Id, files)
+		name, _, err := ce.ProvideFileName(block.GetFile().Name, files, importPath, m.tempDirProvider)
```

Comment:
Here we fix the same problem as from HTML import https://github.com/anyproto/anytype-heart/pull/348
Because Markdown and HTML imports works the same, Markdown import also has a problem with importing local files